### PR TITLE
Display popular skills on members page

### PR DESCRIPTION
**Pull request by robbiecarlton**
Depends on https://github.com/Hylozoic/hylo-node/pull/197/files

### CODE REVIEW

**Comment by robbiecarlton** [comment]
screen shot 2016-12-14 at 7 27 03 pm
screen shot 2016-12-14 at 7 27 21 pm

**Comment by robbiecarlton** [comment]
I think something that's missing here is highlighting the selected skill when it's being searched. I've asked Alex if she has any ideas about that.

File: src/containers/People.js [hunk 1]
```diff
@@ -95,7 +95,12 @@ export default class People extends React.Component {
       total, isMobile, dispatch, cacheId
     } = this.props
     const { currentUser } = this.context
-    if (error) return <AccessErrorMessage error={error}/>
+    if (error) return <AccessErrorMessage error={error} />
+
+    var { popularSkills } = community
+
+    // replacing hyphens with non-breaking hyphens
+    popularSkills = popularSkills.map(skill => skill.replace('-', '‑'))
```
Comment (levity) [inline src/containers/People.js]: nice

File: src/containers/People.js [hunk 2]
```diff
@@ -95,7 +95,12 @@ export default class People extends React.Component {
       total, isMobile, dispatch, cacheId
     } = this.props
     const { currentUser } = this.context
-    if (error) return <AccessErrorMessage error={error}/>
+    if (error) return <AccessErrorMessage error={error} />
+
+    var { popularSkills } = community
```
Comment (levity) [inline src/containers/People.js]: if you combine this and the next line as
```js
import { map } from 'lodash/fp'
const popularSkills = map(skill => skill.replace('-', '‑'), community.popularSkills)
```
then you won't have to create the empty array in the test. in general i think it's good to have the components be forgiving in what data they expect to receive.

**Review by levity — CHANGES_REQUESTED** [review]
Make the one suggested change and we're good

**Pull request merged**